### ISSUE TITLE
Invalid JSON error if exo contains no public folder

### ISSUE THREAD

**Issue by stevennijhof1990** [cisagov/ScubaGear#135]
# 🐛 Summary #

If within EXO no public folders have been defined, the ProviderSettingsExport.json will return two times value within the type

```
        "RootPublicFolderMailbox":  {
                                        "IsValid":  false,
                                        "CanUpdate":  true,
                                        "Type":  {
                                                     "value":  1,
                                                     "Value":  "Unknown"
                                                 },
                                        "HierarchyMailboxGuid":  "00000000-0000-0000-0000-000000000000",
                                        "LockedForMigration":  false
```
As a result, the json cannot be converted

```
ConvertFrom-Json : Cannot convert the JSON string because a dictionary that was converted from the string contains the
duplicated keys 'value' and 'Value'.
At [..]\ScubaGear-main\ScubaGear-main\PowerShell\ScubaGear\Modules\Orchestrator.psm1:207 char:43
+         $TenantDetails = $TenantDetails | ConvertFrom-Json
+                                           ~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [ConvertFrom-Json], InvalidOperationException
    + FullyQualifiedErrorId : DuplicateKeysInJsonString,Microsoft.PowerShell.Commands.ConvertFromJsonCommand
```

## To reproduce ##

1. Ensure that in EXO no Public Folder exist
2. Run the scuba towards exo

## Expected behavior ##

exo output

## Any helpful log output or screenshots ##

See screenshot for exo public folder config and the json convert error
![2023-01-26_11-51-11](https://user-images.githubusercontent.com/123634618/214818348-625b2324-a1f6-4f48-8b94-9df2ffbc1607.png)

**Comment by gdasher** [comment]
We think this issue is a duplicate of issue #132.  we are looking into the best fix there--it seems to be triggered by a Microsoft API change interacting with some strange behavior in ConvertTo-Json.  Closing this as a duplicate, but it is high priority.

**Comment by gdasher** [comment]
We published a release 0.2.1 here: https://github.com/cisagov/ScubaGear/releases/tag/v0.2.1 that should fix this issue (which was due to a microsoft api change that unearthed a bug in the way we handled certain things)
